orchagent/bufferorch.cpp

```diff
@@ -803,6 +805,20 @@ task_process_status BufferOrch::processQueue(KeyOpFieldsValuesTuple &tuple)
                         return handle_status;
                     }
                 }
+                // create/remove a port queue counter for the queue buffer
```

Comment:
as this is used below as well, should you consider having a function to handle the create/remove for queue counter flow?

Reply:
You mean a function in bufferOrch? because there is a dedicated functions in portsOrch for create/remove.